composer.json

```diff
@@ -12,6 +12,8 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.0",
+        "ext-PDO": "*",
```

Comment:
almost there - lower case?

Reply:
Correct php extension name is in uppercase. Above is already tested.

Reply:
> Correct php extension name is in uppercase

What is this based on? Why is pdo upper but pdo_mysql lower case? Could you link the relevant standard?

Reply:
"ext-PDO" is the same case as printed by "php -m" which is the name that the extension author gives that extension. I think php extension names do not follow a naming convention. As you can see in "pcre", "PDO", "Reflection" and "SimpleXML".

Reply:
> I think php extension names do not follow a naming convention

So there is not particular reason for upper case? Then please lower as the other required extensions for sake of consistency.